app/assets/javascripts/form_validations.js

```diff
@@ -42,8 +37,16 @@ var constraints = {
 document.addEventListener('turbolinks:load', function() {
   var forms = document.querySelectorAll('.form');
   if (!forms) return;
-
+  
   forms.forEach(function(form) {
+    
+    form.addEventListener("submit", function(ev) {
+      ev.preventDefault();
+      handleFormSubmit(form);
+      ev.stopPropagation();
```

Comment:
@CouchofTomato I am not sure if this is needed. Why do we need to stop bubbling the event up? I don't think there are event handlers defined on the parent.